packages/main/src/Dialog.ts

```diff
@@ -242,6 +246,8 @@ class Dialog extends Popup {
 
 	/**
 	 * Defines the footer HTML Element.
+	 * <br><br>
+	 * When ui5-bar is used in the Footer additional styling is needed. The default Dialog's paddings should be removed.
```

Comment:
same:
* When a <code>ui5-bar</code> is used in the footer, you should remove the default dialog's paddings.